sse2neon.h

```diff
@@ -2602,6 +2602,21 @@ FORCE_INLINE __m128i _mm_cmpgt_epi8(__m128i a, __m128i b)
         vcgtq_s8(vreinterpretq_s8_m128i(a), vreinterpretq_s8_m128i(b)));
 }
 
+// Compares the 8 signed 16-bit integers in a and the 8 signed 16-bit integers
+// in b for less than.
+//
+//   r0 := (a0 < b0) ? 0xffff : 0x0
+//   r1 := (a1 < b1) ? 0xffff : 0x0
+//   ...
+//   r7 := (a7 < b7) ? 0xffff : 0x0
+//
+// https://technet.microsoft.com/en-us/library/t863edb2(v=vs.100).aspx
```

Comment:
Did you find any valid URL which starts with `msdn.microsoft.com`?

Reply:
My understanding is that all pages on technet and msdn are the same as long as the hash key remains the same:
https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2010/t863edb2(v=vs.100)
works as well as https://technet.microsoft.com/en-us/library/t863edb2(v=vs.100).aspx

Reply:
Thanks for clarification. I was thinking of the feasibility to unify all comments with shorter URLs though.

Reply:
Maybe we can commit the comments change as a separate patch?

Reply:
Can we commit this PR as is, or do I need to change the comment with a different url?

Reply:
Please ignore my previous question. Let's move forward.